mrp_request/migrations/14.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_field_renames = [
+    ("mrp.production", "mrp_production", "mrp_production_request_id", "mrp_request_id"),
+    (
+        "product.template",
+        "product_template",
+        "mrp_production_request_id",
+        "mrp_request_id",
+    ),
+    (
+        "stock.move",
+        "stock_move",
+        "created_mrp_production_request_id",
+        "created_mrp_request_id",
+    ),
+    (
+        "stock.move",
+        "stock_move",
+        "created_mrp_production_request_id",
+        "created_mrp_request_id",
+    ),
+    (
+        "mrp.request.create.mo",
+        "mrp_request_create_mo",
+        "mrp_production_request_id",
+        "mrp_request_id",
+    ),
+    (
+        "mrp.production.request.create.mo.line",
+        "mrp_production_request_create_mo_line",
+        "mrp_production_request_create_mo_id",
+        "mrp_request_create_mo_id",
+    ),
+]
+
+_model_renames = [
+    ("mrp.production.request", "mrp.request"),
+    ("mrp.production.request.create.mo", "mrp.request.create.mo"),
+    ("mrp.production.request.create.mo.line", "mrp.request.create.mo.line"),
+]
+
+_table_renames = [
+    ("mrp_production_request", "mrp_request"),
+    ("mrp_production_request_create_mo", "mrp_request_create_mo"),
+    ("mrp_production_request_create_mo_line", "mrp_request_create_mo_line"),
+]
+
+_xmlid_renames = [
+    (
+        "mrp_production_request.seq_mrp_production_request",
```

Comment:
```suggestion
        "mrp_request.seq_mrp_production_request",
```
because the module will be renamed

do the same to the other xmlids